api_docs/changelog.md

```diff
@@ -20,6 +20,20 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 177**
+
+* [`GET /messages`](/api/get-messages),
+  [`GET /messages/matches_narrow`](/api/check-messages-match-narrow),
+  [`POST /message/flags/narrow`](/api/update-message-flags-for-narrow),
+  [`POST /register`](/api/register-queue):
+  Added support for three [search/narrows](/api/construct-narrow)
+  related to direct messages: `is:dm`, `dm` and `dm-including`.
```

Comment:
nit: I'm not sure how to parse the phrase "three search/narrows related to direct messages". Is "searches/narrows" what you intended?

Per the other comment, though, I'd probably go for something like "three search/narrow filters", linking the phrase "search/narrow filters".

Reply:
I intended to use the same text as the [Changes in Zulip 2.1](https://zulip.com/api/changelog#changes-in-zulip-21) note, but obviously it got mangled as I edited/revised. Updated for your suggestion.